mne/decoding/transformer.py

```diff
@@ -33,20 +33,37 @@ class Scaler(TransformerMixin):
     ----------
     info : instance of Info
         The measurement info
+    scaler : string, defaults to "standard"
+        if scaler == 'robust' sklearn RobustScaler() is used.
+        if scaler == 'standard' applies ``X/std-mean``
+        if scaler == 'default', scales mag by 1e15
+                                       grad by 1e13
+                                       eeg by 1e6
```

Comment:
There shouldn't be any deprecation cycle I think; the original API is equivalent to apply a sklearn.preprocessing.StandardScaler(), isn't?

'default' is a bad name here, because it's actually not the default of the function, but the default parameter used for scalings in MNE.